classes/classes/Scenes/Areas/GlacialRift/Amarok.as

```diff
@@ -137,7 +137,7 @@ package classes.Scenes.Areas.GlacialRift
 			this.tailType = TAIL_TYPE_DOG;
 			this.createStatusEffect(StatusEffects.GenericRunDisabled, 0, 0, 0, 0);
 			//"Watching your movements" alluded to this. Its lore is stalking and hunting people, so I imagine you can't get away.
-			//Otherwise I'd suggest doing a hellhound knock-off of the scent tracking.
```

Comment:
Fixed, thank you for pointing it out!! Had forgotten about that. Would be weird if you had wings but had no chance of escape xD